assertions/confirmation.go

```diff
@@ -83,11 +88,24 @@ func (m *Manager) keepTryingAssertionConfirmation(ctx context.Context, assertion
 			confirmed, err := solimpl.TryConfirmingAssertion(ctx, creationInfo.AssertionHash, prevCreationInfo.ConfirmPeriodBlocks+creationInfo.CreationBlock, m.chain, m.averageTimeForBlockCreation, option.None[protocol.EdgeId]())
 			if err != nil {
 				if !strings.Contains(err.Error(), "PREV_NOT_LATEST_CONFIRMED") {
-					log.Error("Could not confirm assertion", "err", err, "assertionHash", assertionHash.Hash)
+					backoffLogLevel *= 2
+					logLevel := log.Error
+					if backoffLogLevel > time.Minute {
+						backoffLogLevel = time.Minute
+					} else {
+						logLevel = log.Warn
+					}
```

Comment:
seeing the number of times this code block is being repeated, do you think its worth it to incorporate this logic in one place? 

(thinking out loud here) maybe something simpler like having a backoff duration set to ephemeral handler itself and define a new method on it `LogLevelAtCurrentBackOff( time.Duration )` that would return what initial loglevel we want to begin with?

Reply:
I decided to keep it simple and removed the backoff pattern for now.
We can revisit this decision under the logging improvement ticket umbrella later.

Similarly to what was done with StopWaiter, EphemeralErrorHandler was copied from https://github.com/OffchainLabs/nitro.

Since the goal of the ticket related to this PR is simple, I decided to move forward with a straightforward approach, and "took advantage" of the precedence created by the StopWaiter case, and just copied the EphemeralErrorHandler.

In this kind of scenario I usually like to create another git repo, that could be used as a dependency on nitro and bold.
But in previous experiences this type of change created lots of discussions around if it is worth to maintain one more repo, if we should go with a monorepo approach, or if we should just copy and paste, etc.
I wanted to avoid going into this rabbit hole backed up by a change as simple as the one that this PR is trying to achieve, ROI would not be great :(

I think it is a good idea to create a linear ticket related to improving code around logging, not only on bold but also on nitro, and then adding it to triage.
At least nitro relies a lot on a centralized logger, so a change like you are proposing will need a significant effort.
This was discussed once in a team's meeting by the way, and it was decided to postpone changes on the logging pattern.
I raised an issue that, when inspecting log lines of system tests runs that have more than one node, it is not straightforward to identify which node produced a specific log line, and improving that will require a significant effort, mainly due the fact that nitro uses a centralized logging pattern.